orca-dry-run/src/main/kotlin/com/netflix/spinnaker/orca/dryrun/stub/BakeOutputStub.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.dryrun.stub
+
+import com.netflix.spinnaker.orca.pipeline.model.Stage
+import org.springframework.stereotype.Component
+
+@Component
+class BakeOutputStub : OutputStub {
+
+  override fun supports(stageType: String) = stageType == "bake"
+
+  override fun outputs(stage: Stage) =
+    if (stage.parent?.type == "bake") {
+      emptyMap()
+    } else {
+      mapOf(
```

Comment:
We'd need to add a Titus variant here.

Reply:
Were you thinking of putting the cloudprovider specific logic in this single class or break them up into specific classes (my preference)?